.github/workflows/elementor-tests.yml

```
@@ -1,6 +1,6 @@
 name: Elementor Tests / Init
 
-on: [push, pull_request_target]
+on: [pull_request_target]
```

Aligning with other files
```suggestion
on: [pull_request]
```